<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkbenchWidget</class>
 <widget class="QWidget" name="WorkbenchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>工作台</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="groupBoxUser">
     <property name="title">
      <string>作为用户</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QStackedWidget" name="stackedWidgetUser">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageUserEnabled">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButtonUserManageDestination">
            <property name="text">
             <string>管理收货信息</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonUserCreateOrder">
            <property name="text">
             <string>点单</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonUserManageOrder">
            <property name="text">
             <string>查看订单</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonUserManageUser">
            <property name="text">
             <string>管理个人资料</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageUserDisabled">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="labelUserPrompt">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>还不是用户</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonUserRegister">
            <property name="text">
             <string>成为用户</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxCourier">
     <property name="title">
      <string>作为配送员</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QStackedWidget" name="stackedWidgetCourier">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="pageCourierDisabled">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelCourierPrompt">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>还不是配送员</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonCourierRegister">
            <property name="text">
             <string>成为配送员</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageCourierEnabled">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButtonCourierManageOrder">
            <property name="text">
             <string>管理订单</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonCourierManageCourier">
            <property name="text">
             <string>管理配送员资料</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxCanteenManager">
     <property name="title">
      <string>作为餐厅管理员</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QStackedWidget" name="stackedWidgetCanteenManager">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageCanteenManagerEnabled">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QComboBox" name="comboBoxCanteenManager"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonCanteenManagerManageWindow">
            <property name="text">
             <string>管理窗口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonCanteenManagerManageBiz">
            <property name="text">
             <string>管理商户</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonCanteenManagerManageCanteenManager">
            <property name="text">
             <string>管理餐厅管理员</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonCanteenManagerManageCanteen">
            <property name="text">
             <string>管理餐厅信息</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageCanteenManagerDisabled">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="labelCanteenManagerPrompt">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>不是任何餐厅的管理员</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxBizManager">
     <property name="title">
      <string>作为商户管理员</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QStackedWidget" name="stackedWidgetBizManager">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageBizManagerEnabled">
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QComboBox" name="comboBoxBizManager"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonBizManagerManageMeal">
            <property name="text">
             <string>管理菜品</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonBizManagerManageOrder">
            <property name="text">
             <string>管理订单</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonBizManagerManageBizManager">
            <property name="text">
             <string>管理商户管理员</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonBizManagerManageBiz">
            <property name="text">
             <string>管理商户信息</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageBizManagerDisabled">
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="labelBizManagerDisabled">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>不是任何商户的管理员</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxAdmin">
     <property name="title">
      <string>作为超级管理员</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QStackedWidget" name="stackedWidgetAdmin">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageAdminEnabled">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButtonAdminManageAccount">
            <property name="text">
             <string>管理账户</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonAdminManageBiz">
            <property name="text">
             <string>管理商户</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonAdminManageCanteen">
            <property name="text">
             <string>管理食堂</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonAdminManageBizManager">
            <property name="text">
             <string>管理商户管理员</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonAdminManageCanteenManager">
            <property name="text">
             <string>管理食堂管理员</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageAdminDisabled">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="labelAdminPrompt">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>您不是超级管理员</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxAccount">
     <property name="title">
      <string>账号</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButtonAccountManagePhone">
        <property name="text">
         <string>修改手机号</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonAccountManagePassword">
        <property name="text">
         <string>修改密码</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonLogout">
        <property name="text">
         <string>登出</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
